packages/react-components/src/AccessibilityScenarios/Tooltip.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button } from '@fluentui/react-button';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Tooltip } from '@fluentui/react-tooltip';
+
+import { Scenario } from './utils';
+
+export const ButtonsWithTooltipAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Buttons with tooltip">
+      <Tooltip content="Default tooltip behavior">
+        <Button>First</Button>
```

Comment:
For these, I think it'd be good to have something just a bit more like a real-life scenario. We could have a form field with a "more info" button + tooltip (e.g. "Passwords must be at least 6 characters long, and contain XXX"). Another could be a toolbar with a couple icon buttons (used as replacement labels), and a third button with visible text, but a description (maybe something like "Settings"/"Update your profile image and status").

Reply:
I have modified the scenario based on your suggestions.